Source/WebKit/WPEPlatform/wpe/headless/WPEDisplayHeadless.cpp

```diff
@@ -55,23 +55,26 @@
  *
  */
 struct _WPEDisplayHeadlessPrivate {
+    std::optional<WPEDRMDevice*> drmDevice;
```

Comment:
It's redundant to use `std::optional` for a pointer, because we can check whether the pointer is null instead.

Reply:
Not in this case, because we don't know if nullptr means we didn't initialize it or we initialized it but we got nullptr.  So, nullopt means not initialized, nullptr means it's initialized to nullptr.